cdk/setup.py

```diff
@@ -38,7 +38,7 @@
         "aws-cdk.core~=1.187.0",
         "aws-cdk.lambda-layer-awscli~=1.187.0",
         "aws-cdk.lambda-layer-kubectl~=1.187.0",
-        "boto3~=1.26.0",
+        "boto3==1.26.90",
```

Comment:
Updated to fix https://github.com/boto/boto3/issues/3648

Reply:
You can rebase and revert this. I ended up fixing this in a different way that I think is slightly more correct.

Reply:
done